config/checkstyle_sevntu_checks.xml

```diff
@@ -216,5 +216,12 @@
           <property name="methodName" value="assert(True|False)"/>
           <property name="argumentCount" value="1"/>
         </module>
+        <module name="com.github.sevntu.checkstyle.checks.coding.ForbidCertainMethodCheck">
+          <!-- Disallow Locale.setDefault in tests.
+            Use org.junitpioneer.jupiter.DefaultLocale annotation instead.
+            See https://github.com/checkstyle/checkstyle/issues/7436 -->
+          <property name="methodName" value="setDefault"/>
+          <property name="argumentCount" value="1-2"/>
```

Comment:
@romani Are you ok with this?
When I originally suggested this I was under the impression we could specify the class, but this check only signifies the method name which can lead to false positives.

Reply:
http://sevntu-checkstyle.github.io/sevntu.checkstyle/apidocs/com/github/sevntu/checkstyle/checks/coding/ForbidCertainMethodCheck.html
No type ....

It is better to not use such Check for such method name.

Reply:
done